### ISSUE TITLE
missing install rule for orbbec_camera_node

### ISSUE THREAD

**Issue by bmegli** [orbbec/OrbbecSDK_ROS1#22]
The `CMakeLists.txt` currently doesn't install `orbbec_camera_node`

It installs `orbbec_camera_nodelet` only.

**Comment by bmegli** [comment]
Maybe original intention was to
- fill in `EXECUTABLES` variable inside `add_orbbec_executable`

But it doesn't work as expected

https://github.com/orbbec/OrbbecSDK_ROS1/blob/54ca6bcb54ccc731879d4ea155dd9836de8c62f6/CMakeLists.txt#L216-L234

**Comment by bmegli** [comment]
A quick workaround is to add  `${PROJECT_NAME}_node` it to install rules, e.g.

```CMake
# Install
install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_node ${PROJECT_NAME}_nodelet ${EXECUTABLES}
  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
)
```

**Comment by BryanStuurman** [comment]
I have a PR that implements @bmegli 's theory that EXECUTABLES is to grow inside the add_obbec_executable macro, this PR is linked above.

**Comment by bmegli** [comment]
Fixed in 28f0813
